src/MetaMorpho.sol

```diff
@@ -424,27 +417,21 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
     /* REVOKE FUNCTIONS */
 
     /// @inheritdoc IMetaMorphoBase
-    function revokePendingTimelock() external onlyGuardianRole {
-        if (pendingTimelock.validAt == 0) revert ErrorsLib.NoPendingValue();
-
+    function revokePendingTimelockNoRevert() external onlyGuardianRole {
```

Comment:
this naming is very weird from an integrator POV: one may expect this function to never revert but in fact it can revert for any account that doesn't have the guardian role
I understand the motivation but I don't think naming it this way is the way to go, because i don't think it is even useful

Just mention it in the spec of the function and that's enough

Also:
1. `revokePendingMarketRemoval` is not consistent with this naming
2. why not name `acceptCapNoRevert` etc?

Reply:
Agree that this naming isn't the best. I think though that we shouldn't stay with the previous name, because the behavior isn't consistent with the rest of the contracts. What do you think about  ˋremovePendingXIfAny`?

Reply:
I don't like `revokePendingXIfAny` neither, though I prefer it over the current solution

It is verbose to name a function like this and inconsistent with the rest (see my question: why not name `acceptCapNoRevert`?) while the benefit is to point out missing requirements (removed on purpose)

If I'm not mistaken: we're removing the requirement because we don't want to revert in a multicall bundle. It's like when we've removed the role requirement of `acceptX` functions because we want anybody to be able to accept a change, but yet we didn't renamed functions `acceptXPublic` or `acceptXNoRole`

Reply:
In other words, the requirement is removed from this function on purpose, because this function's specs have changed ; but we don't want to name functions so they have the most descriptive and specific name per their spec

Reply:
After thinking a bit I changed my mind you're right. I think there was a strong argument for it when one of the revoke functions did not work like the others (it did not have the same requirement) but now it's ok to leave as is.